### DISCUSSION TITLE
能否按照 group 来展示？

### DISCUSSION THREAD

**Discussion by softwind0214** [zdz/ServerStatus-Rust#232] [General]
这个 group id 好像没体现出来呀，我以为会按照 gid 分组，但它没有。大佬有考虑过这个吗？

**Comment by zdz** [comment]
![image](https://github.com/zdz/ServerStatus-Rust/assets/152173/91a37fcf-c352-4680-b62f-473d2e0ea11e)

目前同一分组的会挨一起，只是界面不明显

**Reply by softwind0214** [reply]
是的，这个我也发现了。建议大佬考虑一下，是不是可以用两种颜色间隔展示

**Reply by zdz** [reply]
主要是整体颜色协调比较麻烦，有空再调

**Reply by softwind0214** [reply]
好的好的，大佬哪天搞了之后踢我一脚
